pyviz_comms/__init__.py

```diff
@@ -49,7 +55,7 @@ class extension(param.ParameterizedFunction):
     def __new__(cls, *args, **kwargs):
         try:
             exec_count = get_ipython().execution_count
-            cls._repeat_execution_in_cell = (exec_count == cls._last_execution_count)
+            extension._repeat_execution_in_cell = (exec_count == cls._last_execution_count)
```

Comment:
This looks like the core of it and this actually looks clearer to me as you don't need to reason about subclasses (where the behavior was wrong anyway)

Reply:
I note that your screenshot example above uses `cls._last_execution_count = exec_count` but in the diff it looks like it was changed to `extension._last_execution_count = exec_count` already.

Reply:
Yes that's right, the screenshot was to show how things worked before the original issue was opened. Issue being fixed in two steps, this PR being the second one.